.gitignore

```diff
@@ -41,3 +41,5 @@ mswindows/osgeo4w/package/
 mswindows/osgeo4w/setup*.hint
 mswindows/osgeo4w/vc/
 daag*.s
+doc/notebooks/.ipynb_checkpoints/
+
```

Comment:
Unnecessary change plus it ends an empty line to the end of the file (there is already a line ending at the end of the file, so you are adding a second one).